src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts

```diff
@@ -91,15 +82,7 @@ namespace OutSystems.OSUI.Utils.PreviewInDevices {
 			}
 		}
 
-		/**
-		 * Adds the required CSS classes, and CSS overrides so that the application adapts correctly
-		 * to the device selected in the preview devices.
-		 *
-		 * @private
-		 * @static
-		 * @param {*} evt
-		 * @memberof OnPostMessage
-		 */
+		// Adds the required CSS classes, and CSS overrides so that the application adapts correctly to the device selected in the preview devices.
```

Comment:
```suggestion
		// Adds the required CSS classes and CSS overrides so that the application adapts correctly to the device selected in the Preview In Devices in ODC
```